.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: "CI"
+
+on:
+  workflow_dispatch:
```

Comment:
do we want to add the option for manually running the workflow? not sure if it is needed for this workflow

Reply:
because realistically one should be able to trigger a workflow from actions on a desired branch anytime without having to push

Reply:
was just wondering because this workflow only runs the tests and lints there is not really a use case in which you would want to run it it manually since whenever you make changes it runs automatically, right?

Reply:
we have the option of manually triggering the workflow in every repo. I think it's still good to be able to rerun workflow on any branch in the repo without having to push the code. I don't think it hurts to keep this here so I want to keep the option.